<launch>
    <node pkg="sc_control" name="sc_control" type="sc_control" output="screen"/>
        <param name="port_name" value="/dev/ttyUSB0"    type="string"/>
        <param name="baud_rate" value="115200"  type="int"/>
</launch>
